client/src/App.vue

```diff
@@ -3,9 +3,10 @@
 </template>
 
 <script>
-    import {DaemonClient} from "daemon_grpc_web_pb"
+
     export default {
         name: 'app'
     };
-    export const daemonclient = new DaemonClient(window.location.protocol+"//"+window.location.hostname+":8000", null, null);
+    export const API_ENDPOINT = window.location.protocol+"//"+window.location.hostname+":8090"
+    export const WEBSOCKET_ENDPOINT = "wss://"+window.location.hostname+":8091"
```

Comment:
Would be nice if it could detect weither or not it should be wss of ws based on if http or https was used. This is mostly since at least i am not using certs when developing locally :)

Reply:
Sure !